samtranslator/model/sam_resources.py

```diff
@@ -1388,15 +1388,13 @@ def to_cloudformation(self, **kwargs):  # type: ignore[no-untyped-def]
             disable_execute_api_endpoint=self.DisableExecuteApiEndpoint,
         )
 
-        (
-            http_api,
-            stage,
-            domain,
-            basepath_mapping,
-            route53,
-        ) = api_generator.to_cloudformation(kwargs.get("route53_record_set_groups", {}))
+        (http_api, stage, domain, basepath_mapping, route53, permissions) = api_generator.to_cloudformation(
```

Comment:
nit: better to keep the previous formatting (add trailing comma), it's a misleadingly large diff for a tiny change